src/client/components/Button/Button.js

```diff
@@ -14,6 +15,7 @@ export default Button;
 Button.propTypes = {
   buttonName: PropTypes.string.isRequired,
   style: PropTypes.string,
+  type: PropTypes.string.isRequired,
```

Comment:
`onClick` is missing `propTypes`

Reply:
If I add proptypes as function here it is also showing the error that don't need. That's why i removed.

Reply:
You should not remove it because it shows an error :) you just need to fix the error

Reply:
I was wrong , I misunderstood the error. now i fixed it.